source/common/http/http2/codec_impl.cc

```diff
@@ -1428,7 +1452,7 @@ Http::Status ServerConnectionImpl::innerDispatch(Buffer::Instance& data) {
   Cleanup cleanup([this]() { dispatching_downstream_data_ = false; });
 
   // Make sure downstream outbound queue was not flooded by the upstream frames.
-  RETURN_IF_ERROR(checkOutboundFrameLimits());
+  RETURN_IF_ERROR(protocol_constraints_.checkOutboundFrameLimits());
```

Comment:
IIUC previously `RETURN_IF_ERROR(checkOutboundFrameLimits());` also checked if `dispatching_downstream_data_` is true. Was it intentionally removed here?

Reply:
The `innerDispatch()` function sets `dispatching_downstream_data_` so it is always `true` in this place.